ginipaybank/src/main/java/net/gini/pay/bank/pay/GiniBank.kt

```diff
@@ -0,0 +1,24 @@
+package net.gini.pay.bank.pay
```

Comment:
I would rename this from `GiniBank` to `GiniBankPay` to reflect that there is no single entry point to this SDK.

`GiniBank` gives the false impression that it's the main class to be use to interact with the SDK.

In comparison to the business SDK which has `GiniBusiness` which clients use as the starting point, this bank SDK can be either used via the `GiniBankCapture` or `GiniBank`. So renaming `GiniBank` to `GiniBankPay` would be clearer.

Reply:
Good point